modules/AllBooks.js

```diff
@@ -0,0 +1,50 @@
+import Book from './Book.js';
+
+export default class AllBooks {
+  constructor() {
+    this.booksSection = document.getElementById('all-books');
+    this.titleInput = document.getElementById('title-input');
+    this.authorInput = document.getElementById('author-input');
+    this.books = [];
+    if (localStorage.getItem('localBooks')) {
+      this.books = JSON.parse(localStorage.getItem('localBooks'));
+      this.displayAllBooks();
+    }
+  }
+
+  formAddBook() {
+    const title = this.titleInput.value;
+    const author = this.authorInput.value;
+    const newBook = new Book(title, author);
+    this.books.push(newBook);
+    this.displayAllBooks();
+    this.storeInLocalStorage();
+    this.titleInput.value = '';
+    this.authorInput.value = '';
+  }
+
+  storeInLocalStorage() {
+    localStorage.setItem('localBooks', JSON.stringify(this.books));
+  }
+
+  displayAllBooks() {
+    this.books.forEach((book, index) => {
+      this.bookDetails = `
+        <div class='book-details'>
+          <h3>${book.title} by ${book.author}</h3>
+          <button class='remove-btn' id='removeBtn'>Remove</button>
+        </div>
+      `;
+      this.booksSection.innerHTML += this.bookDetails;
+      this.deleteBtn = document.getElementById('removeBtn');
+      this.deleteBtn.addEventListener('click', () => this.deleteBook(index));
+    });
+  }
+
+  deleteBook(index) {
+    this.books.splice(index, 1);
+    this.booksSection.innerHTML = '';
+    this.displayAllBooks();
+    this.storeInLocalStorage();
+  }
+}
```

Comment:
- [ ] You did a great job 😊 , but I see that the functionality of removing the book is not working correctly when I click on the remove button for example for the last item the function not working, but when I click on the first remove button it is removing the last item, not the expected item, so please 😊💯 try to fix that to let the Awesome books ready for the production 😃


> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
|     ![Recording 2023-03-28 at 10 25 17](https://user-images.githubusercontent.com/83360636/228175738-e61ce1a9-71e8-4b0a-8121-f13f6ecc7b0a.gif)   |